src/overlays/actors/ovl_En_In/z_en_in.h

```diff
@@ -8,6 +8,30 @@ struct EnIn;
 
 typedef void (*EnInActionFunc)(struct EnIn*, GlobalContext*);
 
+typedef enum {
+    /* 0x00 */ INGO_LIMB_NONE,
+    /* 0x01 */ INGO_LIMB_ROOT,
+    /* 0x02 */ INGO_LEFT_THIGH_LIMB,
+    /* 0x03 */ INGO_LEFT_LEG_LIMB,
```

Comment:
same as en_ma3
```suggestion
    /*  2 */ INGO_LIMB_LEFT_THIGH,
    /*  3 */ INGO_LIMB_LEFT_LEG,
```
you can also use just `L` or `R` for left/right if you want shorter names but I think it's fine as is

do you mean "shin" by "leg"?

Reply:
I had always used thigh, leg, and foot for that.